dts/bindings/reserved-memory/reserved-memory.yaml

```diff
@@ -0,0 +1,23 @@
+description: Reserved memory
+
+compatible: "reserved-memory"
+
```

Comment:
Should add something in the description about `memory-region` references here.

Reply:
`memory-region` is a totally arbitrary phandle. Every driver can use whatever phandle they want to refer to the reserved memory region (using `DT_RESERVED_GET_PTR_BY_PHANDLE`).

Reply:
Not sure when you mean by arbitrary, the binding seems to recommend the usage of that name.  I understand its not required, but I don't see any reason to document that same recommendation (and even possibly codify it by having a "helper" macro with that embedded in it).

(Grepping in linux seems to show `memory-region` used a fair amount as the name)

https://git.kernel.org/pub/scm/linux/kernel/git/devicetree/devicetree-rebasing.git/tree/Bindings/reserved-memory/reserved-memory.txt#n77

Reply:
ok